lib/gollum-lib/markup.rb

```diff
@@ -126,7 +126,7 @@ def render_default(data, format=:markdown, name='render_default.md')
       @format = format
       @name   = name
 
-      chain = [:YAML, :PlainText, :Emoji, :TOC, :RemoteCode, :Code, :Sanitize, :PlantUML, :Tags, :Render]
+      chain = [:YAML, :PlainText, :Emoji, :TOC, :RemoteCode, :Code, :Sanitize, :PlantUML, :Tags, :Render].reject {|filter| skip_filter?(filter)}
```

Comment:
Right, but the `render` and `render_default` methods are enitrely separate from each other, so the rejecting needs to happen twice. But I now notice that `render_default` is called just once in `gollum-lib` and `gollum` taken together, namely in a test. If I modify that test we can just remove this method alltogeter. :+1: